experiment/migrator/migrator.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python2
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Migrates information from Testgrid's Config.yaml to a subdirectory of Prow Jobs
+
+Only moves Dashboard Tabs
+Skips any Dashboard Tab Configuration that contains unusual keys, even if they're incorrect keys.
+"""
+
+import re
+from os import walk
+import ruamel.yaml
+
+PROW_DIR = "../../config/jobs"
+TESTGRID_CONFIG = "../../testgrid/config.yaml"
```

Comment:
nit: consider passing these in as args, eg: https://github.com/kubernetes/test-infra/blob/eb8efa7a9dcdf20840da42a9944bfbdea99c8600/experiment/fix_testgrid_config.py#L90-L99